src/components/BounceDetailsContainer/Modals/ConfirmationModal/index.jsx

```diff
@@ -41,6 +43,16 @@ const ConfirmModalBody = ({
             {isUpdateError && (
               <UpdateAlertError handleModalClose={handleModalClose} />
             )}
+            {isCommitEmpty && (
+              <Alert
```

Comment:
can you make the required part be inline validation (https://sendgrid.github.io/ui-components/?selectedKind=Text%20Input&selectedStory=Text%20Input%20With%20Error&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)? and disable the "Confirm" button until a commit message has been entered. we can keep the alerts, but only show them if after the user confirms and the server returns an error (display the server error in this alert banner). there's also a red dot that indicates required you can add (https://sendgrid.github.io/ui-components/?selectedKind=Text%20Input&selectedStory=Text%20Input%20Simple%20with%20a%20value&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

Reply:
and for the inline error message, use "This field is required." to be consistent with what we have elsewhere on SendGrid